sorc/checkout.sh

```diff
@@ -55,7 +55,7 @@ if [[ ! -d gldas.fd ]] ; then
     rm -f ${topdir}/checkout-gldas.log
     git clone https://github.com/NOAA-EMC/GLDAS.git gldas.fd >> ${logdir}/checkout-gldas.fd.log 2>&1
     cd gldas.fd
-    git checkout gldas_gfsv16_release.v1.15.0
+    git checkout gldas_gfsv16_release.v.1.28.0
```

Comment:
I assume this tag works with current `develop` of the workflow.
I recall this is the same tag as in operations headed to WCOSS2.

Reply:
> Changes look good. I assume this was tested beyond just building. Especially utilities from UFS_utils.

I hadn't because I was focused on the original problem, which was just a bad ESMF module load and module reset not working on other machines. But since these are version updates, you're probably right and I should run a test.

Reply:
It is. Kate got NCO to update the version for the handoff to include the tag that had the module fixes for other machines.